code/modules/mob/living/simple_animal/hostile/zombie.dm

```diff
@@ -1,184 +1,153 @@
 /mob/living/simple_animal/hostile/zombie
 	name = "zombie"
-	desc = "A horrifying, shambling carcass of what once resembled a human being."
+	desc = "When Observe is full, the dead shall walk the station."
 	icon = 'icons/mob/human.dmi'
 	icon_state = "zombie_s"
 	icon_living = "zombie_s"
 	icon_dead = "zombie_dead"
 	turns_per_move = 5
-	emote_see = list("groans", "roars")
+	speak_emote = list("groans")
+	emote_see = list("groans")
 	a_intent = "harm"
 	maxHealth = 120
 	health = 120
 	speed = 2
-	move_to_delay = 5
 	harm_intent_damage = 8
-	melee_damage_lower = 9
+	melee_damage_lower = 20
```

Comment:
Why did you up melee_damage_lower? Now zombies will consistently dish out 20 damage per attack. Is that what you want?